src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBaseKernels.hpp

```diff
@@ -566,6 +566,58 @@ struct SolutionCheckKernel
 
 };
 
+/******************************** StatisticsKernel ********************************/
+
+struct StatisticsKernel
+{
+  template< typename POLICY >
+  static void
+  launch( localIndex const size,
+          arrayView1d< integer const > const & elemGhostRank,
+          arrayView1d< real64 const > const & volume,
+          arrayView1d< real64 const > const & pres,
+          arrayView1d< real64 const > const & refPorosity,
+          arrayView2d< real64 const > const & porosity,
+          real64 & minPres,
+          real64 & avgPresNumerator,
+          real64 & maxPres,
+          real64 & totalUncompactedPoreVol,
+          real64 & totalPoreVol )
+  {
+    RAJA::ReduceMin< parallelDeviceReduce, real64 > subRegionMinPres( LvArray::NumericLimits< real64 >::max );
+    RAJA::ReduceSum< parallelDeviceReduce, real64 > subRegionAvgPresNumerator( 0.0 );
+    RAJA::ReduceMax< parallelDeviceReduce, real64 > subRegionMaxPres( 0.0 );
```

Comment:
This is meant to be positive or can you have negative values too?
In the latter case, starting value `0.0` needs to be changed.
And in the first case maybe too (do we intend this value to be unchanged?)...

Reply:
You are right, I changed the min value initialization to `-LvArray::NumericLimits< real64 >::max` because the single-phase solver allows for negative values.